src/code/z_camera.c

```diff
@@ -84,13 +99,78 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_camera/func_800CC5C8.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_camera/func_800CC740.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_camera/func_800CC7A8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_camera/func_800CC804.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_camera/func_800CC874.s")
+#define CAM_DATA_IS_BG 0x1000 // if not set, then cam data is for actor cutscenes
```

Comment:
```suggestion
#define CAM_DATA_IS_BG (1 << 12) // if not set, then cam data is for actor cutscenes
```
This is a flag right?